README.md

```diff
@@ -102,11 +102,26 @@ Bump new version number in `CHANGELOG.md` and `README.md`.
 Commit, tag and create a new release:
 
 ```sh
+version="v1.6.0"
+
+git switch --create release/${version}
 git add CHANGELOG.md README.md
-git commit -m "Bump v1.5.0"
-git tag v1.5.0
-git push origin master v1.5.0
-gh release create v1.5.0
+git commit -m "Bump ${version}"
+git push --set-upstream origin release/${version}
+gh pr create --reviewer=EtienneM --title "$(git log -1 --pretty=%B)"
+```
+
+Once the pull request merged, you can tag the new release.
+
+### Tag the New Release
+
+```bash
+git tag v${version}
+git push origin master v${version}
+```
+
+git push origin master v1.6.0
+gh release create v1.6.0
```

Comment:
There is a problem here, I'm not sure about what you eventually wanted to have

Reply:
Oups a bad copy / paste